packages/k8s.contrib.crd/internal/ModuleGenerator.pkl

```diff
@@ -102,7 +110,7 @@ converters: Mapping<List<String>, Module|Class|TypeAlias>
 baseUri: URI
 
 /// The name of this module
-moduleName: String = crd.spec.names.kind
+moduleName: String = crd.spec.group.split(".").reverse().add(crd.spec.names.kind).join(".")
```

Comment:
Our convention for k8s is to use the reverse-FQDN of the group, then the module name.